Sources/VSM/ViewState/RenderedViewState.swift

```diff
@@ -177,10 +178,19 @@ extension RenderedViewState.RenderedContainer: StateObserving & StatePublishing
     // MARK: StatePublishing
     // For more information about these members, view the protocol definition
     
+    @available(*, deprecated, renamed: "didSetPublisher", message: "It has been renamed to didSetPublisher and will be removed in a future version.")
```

Comment:
nit - just say "Renamed to didSetPublisher and will be removed in a future version."
"It has been" is ambiguous and passive tense.